snp_blending/config/workcell.srdf

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" ?>
+<robot name="motoman_mh_180_120" version="1.0.0">
+
+    <group name="manipulator">
+        <chain base_link="base_link" tip_link="tool0"/>
+    </group>
+
+    <!--Groups kinematic plugins -->
+    <kinematics_plugin_config filename="package://snp_blending/config/workcell_plugins.yaml"/>
+
+    <!-- Contact manager plugins -->
+    <contact_managers_plugin_config filename="package://snp_blending/config/workcell_plugins.yaml"/>
+
+    <group_opw group="manipulator"
+    a1="0.325" a2="-0.3" b="0" c1="0.65" c2="1.15" c3="1.590" c4="0.225"
+    offsets="0.0 0.0 -1.570796 0.0 0.0 0.0"
+    sign_corrections="1 1 -1 -1 -1 -1"/>
```

Comment:
This can be removed now; it's captured in the OPW parameters in `workcell_plugins.yaml`